### PR TITLE
Allow multiple array values for actions

### PR DESCRIPTION
**Pull request by paramite**
Plugin omelasticsearch allows array value for host configuration, which is not possible to pass with this module. This patch adds that possibility.

### CODE REVIEW

File: spec/acceptance/actions_spec.rb [hunk 1]
```diff
@@ -25,6 +25,7 @@ class { 'rsyslog::config':
           'myaction' => {
             'type'   => 'omelasticsearch',
             'config' => {
+              'host'                 => '["127.0.0.1", "192.168.0.1"]',
```
Comment (bastelfreak) [inline spec/acceptance/actions_spec.rb:28]: Is there a reason why the array is actually a string? can't we use a real array datatype here?

Comment (paramite) [reply spec/acceptance/actions_spec.rb:28]: No reason. It works with array too.

**Comment by paramite** [comment]
Hmm, ok it worked for me locally :). Will resolve the failures.

File: templates/action.epp [hunk 2]
```diff
@@ -26,7 +26,11 @@ action(type="<%= $type %>"
 <%}-%>
 <% if $config { -%>
   <% $config.each |$k, $v| { -%>
+  <%- if is_array($v) { -%>
```
Comment (bastelfreak) [inline templates/action.epp:29]: the `is_array()` method is old, deprecated, Puppet has a built-in matcher:
```suggestion
  <%- if  $v =~ Array { -%>
```

Comment (paramite) [reply templates/action.epp]: Thanks!

Comment (bastelfreak) [reply templates/action.epp:29]: did you switch back to the old syntax?

**Comment by paramite** [comment]
\o/

**Comment by paramite** [comment]
@bastelfreak Is there anything else I should do?

**Pull request merged**